test/jb_splitsStore/set.test.js

```diff
@@ -0,0 +1,293 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { deployMockContract } from '@ethereum-waffle/mock-contract';
+import { abi as jbOperatorStoreAbi } from '../../artifacts/contracts/JBOperatorStore.sol/JBOperatorStore.json';
+import { abi as jbProjectsAbi } from '../../artifacts/contracts/JBProjects.sol/JBProjects.json';
+import { abi as jbDirectoryAbi } from '../../artifacts/contracts/JBDirectory.sol/JBDirectory.json';
+
+const ZERO_ADDRESS = ethers.utils.hexZeroPad('0x', 20); // address(0)
+const PROJECT_ID = 1;
+const DOMAIN = 2;
+const GROUP = 3;
+let SET_SPLITS;
+
+describe('JBSplitsStore::set(...)', function () {
+
+  before(async function () {
+    let jbOperationsFactory = await ethers.getContractFactory('JBOperations');
+    let jbOperations = await jbOperationsFactory.deploy();
+
+    SET_SPLITS = await jbOperations.SET_SPLITS();
+  });
+
+
+  async function setup() {
+    let [deployer, projectOwner, ...addrs] = await ethers.getSigners();
+
+    // Override authentification (see jb_operator_store for auth tests)
+    let mockOperatorStore = await deployMockContract(deployer, jbOperatorStoreAbi);
+    let mockProjects = await deployMockContract(deployer, jbProjectsAbi);
+    let mockDirectory = await deployMockContract(deployer, jbDirectoryAbi);
+
+    // Override auth: by default projectOwner is controller + owner + has permission on SET_SPLITS
+    await mockOperatorStore.mock.hasPermission
+      .withArgs(projectOwner.address, projectOwner.address, PROJECT_ID, SET_SPLITS)
+      .returns(true);
+
+    await mockProjects.mock.ownerOf
+      .withArgs(PROJECT_ID)
+      .returns(projectOwner.address);
+
+    await mockDirectory.mock.controllerOf
+      .withArgs(PROJECT_ID)
+      .returns(projectOwner.address);
+
+    let jbSplitsStoreFact = await ethers.getContractFactory('JBSplitsStore');
+    let jbSplitsStore = await jbSplitsStoreFact.deploy(
+      /*operator=*/mockOperatorStore.address,
+      /*projects=*/mockProjects.address,
+      /*directory=*/mockDirectory.address
+    );
+
+    let splits = createSplitArray(addrs[0].address, 4);
+
+    return { deployer, projectOwner, addrs, jbSplitsStore, splits, mockOperatorStore, mockProjects, mockDirectory };
+  }
+
+  // Create array of JBSplit struct
+  function createSplitArray(_benefAdress, n) {
```

Comment:
Remove underscore-prefixed benefAddress - should just be regulator camel case.

Let's also not abbreviate – should it be beneficiaryAddress?